### PR TITLE
RAC-1096: Merge filter default value and local stored value

### PR DESCRIPTION
**Pull request by ghoussard**
**Description (for Contributor and Core Developer)**

To avoid blank page due to a error occurred when a new filter is not in local stored filter value, we need to merge default filter value with stored filter value.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx [hunk 1]
```diff
@@ -12,23 +11,15 @@ import {
...
 const JobExecutionList = () => {
   const activityHref = useRoute('pim_dashboard_index');
   const translate = useTranslate();
-  const [jobExecutionFilter, setJobExecutionFilter] = useStorageState<JobExecutionFilter>(
-    getDefaultJobExecutionFilter(),
-    FILTER_LOCAL_STORAGE_KEY
-  );
+
+  const [jobExecutionFilter, setJobExecutionFilter] = useStoredJobExecutionFilter();
+  console.log(jobExecutionFilter);
```
Comment (pierrejolly) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx:22]: to remove

Comment (ValentinMumble) [reply src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx]: To remove

**Review by ValentinMumble — APPROVED** [review]

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx (hunk 1 above)
Comment (ghoussard) [reply src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx]: To remove

Comment (StevenVAIDIE) [reply src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx]: To remove

**Review by StevenVAIDIE — APPROVED** [review]

**Pull request merged**